SDKKeyStore.swift

```diff
@@ -0,0 +1,110 @@
+import Foundation
+
+@objc public class SDKKeyStore: NSObject {
+    let service: String
+    var account: String
+
+    typealias KeysDictionary = [String: String]
+
+    init(service: String = Bundle.main.bundleIdentifier!) {
+        self.service = service
+        self.account = "\(service).keys"
```

Comment:
I wonder if this is specific enough. What if the app already has a keychain store called `<bundle id>.keys` - maybe it should be `.rakuten.tech.sdkkeys` to avoid potential clashes?

Reply:
I've added this.